include/sdbus-c++/ConvenienceApiClasses.inl

```diff
@@ -485,15 +485,16 @@ namespace sdbus {
     }
 
 
-    inline AsyncMethodInvoker::AsyncMethodInvoker(IProxy& proxy, const std::string& methodName)
+    inline AsyncMethodInvoker::AsyncMethodInvoker(IProxy& proxy, const std::string& methodName, uint64_t timeout)
```

Comment:
Support for D-Bus call timeout makes a whole lot of sense also for classic sync calls. Would you be so kind to also extend the API of sync calls to support custom timeouts?